wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -1,406 +1,355 @@
-
-<table>
-<thead>
-<tr class="header">
-<th><p>Hit Object Timeline</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>&quot;+&quot;/&quot;-&quot; buttons</p></td>
-</tr>
-<tr class="even">
-<td><p>Double white vertical lines</p></td>
-</tr>
-<tr class="odd">
-<td><p>Click on hitobject</p></td>
-</tr>
-</tbody>
-</table>
-
-==Top-Right (Beat Snap Divisor)== <img src="BE_BSD.jpg" title="fig:Beat Snap Divisor" alt="Beat Snap Divisor" width="200" />
+### Top-Left (Hit Objects Timeline)
+
+![Hit Objects Timeline](/wiki/shared/BE_NTL.jpg "Hit Objects Timeline")
+
+**Shows the timeline for hitobjects with respect to Beat Snap Divisor and timestamp.**
+
+#### Hit Object Timeline
+
+| Name | Description |
+| ---- | ----------- |
+| "+"/"-" buttons | Increase/Decrease timeline zoom |
+| Double white vertical lines | Current timestamp with respect to the Hit Objects Timeline. |
+| Click on hitobject | **Left-Click:** Select / Move hitobject against timeline. **Right-Click:** Remove hitobject. |
+
+### Top-Right (Beat Snap Divisor)
+
+![Beat Snap Divisor](BE_BSD.jpg "Beat Snap Divisor")
 
 **In layman term, adjusting given length of a coloured sewing thread (timing section) for an embroidery design space (measures, divided by the divisor to be called beats) on the cloth (song).**
 
 Hold **Alt** to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
-| A Measure                        | Divisor | Beat              | Ticks notes                                        | Notes                                                      |
-|----------------------------------|---------|-------------------|----------------------------------------------------|------------------------------------------------------------|
-| ![](BSD 1 1b.jpg "BSD 1 1b.jpg") | 1/1     | Full Beats        | White ticks only                                   | Best used for Easy difficulty.                             |
-| ![](BSD 1 2.jpg "BSD 1 2.jpg")   | 1/2     | Half Beats        | Adds red ticks                                     | Best used for Easy/Normal difficulties.                    |
-| ![](BSD 1 3.jpg "BSD 1 3.jpg")   | 1/3     | Triplets\*        | Has two purple ticks between each white tick       | Compound timing - songs which use this are rare.           |
-| ![](BSD 1 4.jpg "BSD 1 4.jpg")   | 1/4     | Quarter Beats     | Adds blue ticks, in addition to the red ticks      | Preference for general mapping, no matter what difficulty. |
-| ![](BSD 1 6.jpg "BSD 1 6.jpg")   | 1/6     | Double Triplets\* | Has two purple ticks between each red tick         | Compound timing - same as 1/3.                             |
-| ![](BSD 1 8.jpg "BSD 1 8.jpg")   | 1/8     | Eighth Beats      | Adds yellow ticks, in addition to the red and blue | Not recommended unless you are very experienced.           |
+#### Divisor 1/1
```

Comment:
Is this (and below) just a dup of the `Beat_Snap_Divisor`? If so, just link to it with something like `_See also [Beat Snap Divisor](../Beat_Snap_Divisor)_`